src/server/snap.rs

```diff
@@ -299,17 +315,40 @@ impl<R: RaftStoreRouter + 'static> Runnable<Task> for Runner<R> {
                 }
             }
             Task::SendTo { addr, msg, cb } => {
+                if self.sending_count.load(Ordering::Acquire) >= MAX_SENDER_CONCURRENT {
```

Comment:
Use `SeqCst` instead.

Reply:
I think `Acquire` and `Release` pair is more relaxed than `SeqCst`?